### PR TITLE
TC_15.001.005.001 | Teacher Profile > Hiding announcement

### PR DESCRIPTION
**Pull request by podko-va**
[TC_15.001.005.001 | Teacher Profile > Hiding announcement > Name changes and teacher's announcement became invisibile.](https://github.com/orgs/RedRoverSchool/projects/2/views/2?filterQuery=15.001&pane=issue&itemId=91157732&issue=RedRoverSchool%7CBookClubQA_Python_2024_fall%7C265)

### CODE REVIEW

**Comment by OxanaSum** [comment]
Нужно прикрепить ссылку на тест-кейс с борда. Можно в 1-м сообщении

**Comment by podko-va** [comment]
[TC_15.001.005.001 | Teacher Profile > Hiding announcement > Name changes and teacher's announcement became invisibile.](https://github.com/orgs/RedRoverSchool/projects/2/views/2?filterQuery=15.001&pane=issue&itemId=91157732&issue=RedRoverSchool%7CBookClubQA_Python_2024_fall%7C265)

File: tests/test_announcement.py [hunk 1]
```diff
@@ -40,6 +44,43 @@ def test_teacher_announcement_blank_form(
...
+        announcement.click_make_announcement_visible()
+        announcement.check_button_text_visible()
+        
+        try:
+            announcement.check_teacher_announcement_invisible()
+        except Exception as e:
+            allure.attach(
+                str(e),
+                name="Error in check_teacher_announcement_invisible",
+                attachment_type=allure.attachment_type.TEXT
```
Comment (KatyaMy) [inline tests/test_announcement.py:64]: это тебе для логов в случае чего? У нас есть скрин и видео собирает. Нужен ли он тут

**Review by KatyaMy — APPROVED** [review]

File: tests/test_announcement.py (hunk 1 above)
Comment (podko-va) [reply tests/test_announcement.py]: все верно убрала

**Pull request merged**